woodwork/tests/type_system/test_ltype_inference.py

```diff
@@ -103,20 +103,15 @@ def test_email_inference(emails):
 
 
 def test_email_inference_failure(bad_emails):
-    dtypes = ["object", "string"]
-    if _is_koalas_series(bad_emails[0]):
-        dtypes = get_koalas_dtypes(dtypes)
-
     for series in bad_emails:
-        for dtype in dtypes:
-            inferred_type = ww.type_system.infer_logical_type(series.astype(dtype))
-            assert not isinstance(inferred_type, EmailAddress)
```

Comment:
Is there a reason this was changed so that we aren't testing with multiple input dtypes now?

Reply:
In the new spark version, we can't convert a series of dictionaries into an object or string type

```python
bad_email = pd.Series([{"key": "value"}]).astype("O")
ps.from_pandas(bad_email).astype('string')
```
```
TypeError: astype can not be applied to structs.
```

Reply:
I see. Could we just skip that particular series for pyspark, but leave the test the same otherwise for pandas/Dask?

Reply:
updated in c5ec0399000c3415201d868ba68c5e84be85b09b